x/rollapp/keeper/liveness_test.go

```diff
@@ -74,21 +76,18 @@ func TestLivenessEventsStorage(t *testing.T) {
 }
 
 // The protocol works.
-func TestLivenessFlow(t *testing.T) {
+func (suite *RollappTestSuite) TestLivenessFlow() {
 	_ = flag.Set("rapid.checks", "500")
 	_ = flag.Set("rapid.steps", "300")
-	rapid.Check(t, func(r *rapid.T) {
-		s := new(RollappTestSuite)
-		s.SetT(t)
-		s.SetS(s)
-		s.SetupTest()
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
 
```

Comment:
prefer change other receivers to `s` 
long receiver names are not go idiomatic

Reply:
I prefer `suite`